src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid $white-border-color;
+  border-radius: $card-border-radius;
+  width: $card-width;
+
+  &__info-logo {
+    width: 160px;
+    height: 134px;
+    padding: 0 3px;
+    margin-bottom: 40px;
+  }
+
+  &__info-title {
+    color: $dark-color;
+    font-family: $font-family-inherit;
```

Comment:
you already set a font-family in html tag? Is this really necessary to add a variable few times during your code?